chart/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
+.DS_Store
+# Common VCS dirs
+.git/
+.gitignore
+.bzr/
+.bzrignore
+.hg/
+.hgignore
+.svn/
+# Common backup files
+*.swp
+*.bak
+*.tmp
+*.orig
+*~
+# Various IDEs
+.project
+.idea/
+*.tmproj
+.vscode/
```

Comment:
For .gitignore a rule of thumb is that various IDEs, .DS_Store etc. should be in a users system-wide config rather than being repeated within individual projects. Is that an option for Helm as well?
Ideally we'd only explicitly ignore things that are part of this repo (or created by scripts within it).

Reply:
I updated its content as requested by just ignore `doc/` which is not required during `helm package` process